index.html

```diff
@@ -1779,115 +1829,218 @@ <h3>Anonymous Holder Binding</h3>
-conveyed to the holder as part of the base proof value.
+Anonymous holder binding and credential-bound pseudonyms are, in a sense,
+orthogonal features, and a holder and credential ecosystem may wish to use both
+at the same time. For instance, a holder may wish to bind a verifiable
+credential to a |holder_secret|, so that only a holder knowing this value can
+generate a derived proof from the base proof, and bind a |nym_secret| to the
+base proof so that pseudonyms can be bound to the derived proof. This
+corresponds to the |featureOption| being equal to `"holder_binding_pseudonym"`.
         </p>
         <p>
-The holder obtains a |verifier_id| associated with the verifier for which
-they intend to create a revealed document with derived proof. Using the
-procedures of [[CFRG-Pseudonym-BBS-Signature]], a cryptographic |pseudonym|
-value is generated from their |pid| value and the |verifier_id|. The derived
-proof value is generated via the proof generation using the |pid|,
-|secret_prover_blind|, |verifier_id|, and |signer_blind| using the
-procedures of [[CFRG-Pseudonym-BBS-Signature]],
-and this value is given to the
-verifier along with the |pseudonym|. Note that the |pid| value cannot be
-recovered from the |pseudonym|.
-        </p>
-        <p>
-When the verifier receives the revealed document with derived proof and
-|pseudonym|, they use the proof verification procedures of
-[[CFRG-Pseudonym-BBS-Signature]].
+An outline of the creation and use of both anonymous holder binding and
+credential bound pseudonyms is given in the following steps.
         </p>
+        <ol class="algorithm">
+          <li>
+The holder creates two secure values known as the |holder_secret| and
+|prover_nym| that should be at
+least 32 bytes long and generated in cryptographically random manner. They then
+use the "Commitment" operation from [[CFRG-Pseudonym-BBS-Signature]] with the
+|committed_messages| array input containing the |holder_secret| as its only
+element, to compute the |commitment_with_proof| and |secret_prover_blind|. The
+|commitment_with_proof| is sent with their request to the issuer for a
+credential while the |holder_secret|, |prover_nym| and |secret_prover_blind| are
+kept for later use by the holder and are never disclosed.
+          </li>
+          <li>
+The issuer receives the holders request for a credential with a bound pseudonym,
+|featureOption| equals `"pseudonym"`, along with the |commitment_with_proof|.
+The issuer uses the procedures in this specification and generates a
+cryptographically random value for the |signer_nym_entropy| and uses the
+"Blind Issuance" operation from [[CFRG-Pseudonym-BBS-Signature]] to produce the
+base proof. Among other information the base proof will contain the
+|signer_nym_entropy| value. If the issuer ever needs reissue a credential to
+this holder that is bound to the same |nym_secret| they should retain the
+|signer_nym_entropy| value, otherwise this value can be discarded.
```

Comment:
```suggestion
|signer_nym_entropy| value. If the issuer ever needs reissue a credential to
this holder that is bound to the same |prover_nym| they should retain the
|signer_nym_entropy| value, otherwise this value can be discarded.
```

There is no `nym_secret`